cruft/_commands/update.py

```diff
@@ -144,8 +144,24 @@ def _apply_patch_with_rejections(diff: str, expanded_dir_path: Path):
 
 def _apply_three_way_patch(diff: str, expanded_dir_path: Path):
     try:
+        offset = (
+            run(
+                ["git", "rev-parse", "--show-prefix"],
+                stderr=PIPE,
+                stdout=PIPE,
+                check=True,
+                cwd=expanded_dir_path,
+            )
+            .stdout.decode()
+            .strip()
+        )
+
+        git_apply = ["git", "apply", "-3"]
+        if len(offset) > 0:
```

Comment:
```suggestion
        if offset:
```

Reply:
Also update apply patch with rejections with the offset code?

Reply:
@samj1912 something was odd on this PR. Updated the code.

Reply:
You mean _apply_patch_with_rejections?

Reply:
> You mean _apply_patch_with_rejections?

Yup! :)

Reply:
Can you also make this change please :)